tests/virtualization/xen/virtmanager_offon.pm

```diff
@@ -51,7 +62,7 @@ sub run {
 }
 
 sub test_flags {
-    return {fatal => 1, milestone => 0};
+    return {fatal => 0, milestone => 1};
```

Comment:
When test is milestone it makes sense to be also fatal, going from bottom to top ^^, same as below